packages/Notification/Notification.md

```diff
@@ -64,3 +64,36 @@ Use the `dismissible` prop to allow users to dismiss the `Notification` at any t
   confirmation email sent after your purchase.
 </Notification>
 ```
+
+### Using Callbacks
+
+#### onDismiss
+
+Use the `onDismiss` prop to execute a function when a user dismisses the `Notification`. The `onDismiss` prop must be used together with the `dismissible` and `dismissibleA11yLabel` props.
```

Comment:
I know it's already in the prop descriptions but I think we should add the same info here about what it means to be dismissed vs exited.

Ex. that `onDismiss` is on unmount and `onExit` is on click